Parser/pegen/pegen.c

```diff
@@ -720,6 +720,26 @@ seq_insert_in_front(Parser *p, void *a, asdl_seq *seq)
     return new_seq;
 }
 
+/* Creates a copy of seq and prepends a to it */
+asdl_seq *
+concatenate_sequences(Parser *p, asdl_seq *a, asdl_seq *b)
```

Comment:
There's already the private function `_join_seqs`, which you can use.

Reply:
Do we need this? Couldn't we just make either one of these public and just use that one?

Reply:
Good point. Sorry, I had my mind on the error messages and fixed this too quickly :S

Reply:
No worries!